polling_stations/apps/data_finder/views.py

```diff
@@ -286,7 +255,7 @@ def get_location(self):
         )
 
     def get_council(self, geocode_result):
-        return Council.objects.defer("area").get(identifiers__contains=["E06000023"])
+        return Council.objects.defer("area").get(pk="E06000023")
```

Comment:
This looks like the fixture should be changed so it doesn't look like a gss code.